src/libs/Navigation/AppNavigator/BaseDrawerNavigator.js

```diff
@@ -3,8 +3,8 @@ import _ from 'underscore';
 import PropTypes from 'prop-types';
 import {createDrawerNavigator} from '@react-navigation/drawer';
 import {View} from 'react-native';
-import styles, {getNavigationDrawerStyle, getNavigationDrawerType} from '../../../styles/styles';
 import withWindowDimensions, {windowDimensionsPropTypes} from '../../../components/withWindowDimensions';
+import styles, {getNavigationDrawerStyle, getNavigationDrawerType} from '../../../styles/styles';
```

Comment:
Why move this down here? I kinda like it better above `withWindowDimensions` to make imports slightly more alphabetical

Reply:
Not a big deal, I'd rather get this merged haha

Reply:
Oh, what I am 9 minutes late.... :panda_face:

Reply:
lol it's fine - i thought you would be later :D